docs/rfc/rfc-8.adoc

```diff
@@ -0,0 +1,79 @@
+:toc: macro
+
+= RFC 8: Cross-chain Tokenized Threshold BTC
+
+:icons: font
+:numbered:
+:stem:
+toc::[]
+
+== Goal
+
+The goal of this proposal is to enable *efficient* cross-chain tBTC liquidity.
+
+In the Ethereum ecosystem, that means bringing Bitcoin to L2s and sidechains like
+Arbitrum, Optimism, Polygon, Avalanche, and Binance Smart Chain. In the Cosmos
+ecosystem, that means Bitcoin on IBC-enabled zones (chains) like Osmosis and
+Penumbra. And in newer ecosystems, that means chains like Solana, Aptos, and
+Sui.
+
+Efficiency requires minimizing liquidity fragmentation by avoiding the need for
+tBTC wrapper tokens by each chain and bridge. This situation quickly leads to an
+stem:[n*m] problem, where liquidity on `n` chains is divided between `m`
+bridges. Each of these token variants soaks up valuable liquidity that could be
+better utilized elsehwere.
+
+Because tBTC is minted "centrally" on Ethereum L1, we can do better. By
```

Comment:
Does it mean that we could surpass L1 and mint directly from BTC to ETH L2? instead of BTC-->L1--->L2 just have BTC --->L2

Reply:
This proposal assumes TBTC is minted on L1 first, always.

There are ways to mint directly on L2 but this is something for the future as it is quite complex and requires remote mind and burn implementation for L1<>L2 bridging.